README.md

```diff
@@ -1,5 +1,25 @@
 # Student Explorer
 
+## Overview
+
+Student Explorer is an early warning system that helps academic advisors at the University of Michigan-Ann Arbor identify at-risk students based on current term Learning Management System data. It started in 2011 as a research project at the University of Michigan USE Lab; LSA Technology Services, Academic Innovation, and ITS Teaching & Learning have contributed to the design, development, and management of the tool. Student Explorer is in use as an enterprise production application at U-M and continues to be developed by ITS Teaching & Learning. The code is open source and is licensed under Apache 2.0. The application primarily relies on Python Django for the backend and Bootstrap and jQuery for the frontend.
```

Comment:
"primarily relies on Python and Django"?

Reply:
I'm not entirely sure what your question means but I have a feeling you misread it as "Python and Django" but it says "Python Django" but this indicates to me that it's confusing and it should just say "Django." Thoughts?

Reply:
Sorry, I was suggesting changing it to "Python and Django". Or just "Django", since it implies Python. "Python Django" didn't make sense to me.

Reply:
I'm feeling just Django, as you said it implies Python. I think "Python and Django" doesn't make perfect sense because Django is Python? It's like saying "Rectangle and Square". Will add a commit.